frontend/packages/dev-console/src/utils/icons.tsx

```diff
@@ -6,19 +6,19 @@ import {
   LaptopCodeIcon,
   OsImageIcon,
   FileUploadIcon,
+  GitAltIcon,
+  OutlinedFileCodeIcon,
 } from '@patternfly/react-icons';
 import * as devfileIcon from '../images/devfile.svg';
 import * as dockerfileIcon from '../images/dockerfile.svg';
-import * as gitIcon from '../images/from-git.svg';
-import * as yamlIcon from '../images/yaml.svg';
 
-export const gitIconSVG = gitIcon;
+export const gitIconElement = <GitAltIcon />;
 
 export const devfileIconSVG = devfileIcon;
 
 export const dockerfileIconSVG = dockerfileIcon;
```

Comment:
We can also remove these two unused variables, I was surprised that all were components now but these are not.

```suggestion

export const devfileIconSVG = devfileIcon;

export const dockerfileIconSVG = dockerfileIcon;
```